demo/src/app/components/typeahead/overview/sections/user-interaction/template.html

```diff
@@ -0,0 +1,82 @@
+<h5>Edit the search term</h5>
```

Comment:
I see that it might go too much into the obvious (for instance, I could go deeper and say that to edit they have to use anything that throws keys events to the user agent, bla bla bla that would be too much). Having an `input` element makes it quite obvious.

However, I could maybe remind and say more precisely that "with a classic usage" (I don't know how to turn that yet), the search is done anytime the input value changes, i.e. vs not only when blurring or using a special key.

That's part of the user interaction with Typeahead, so I would keep that.